app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
```

Comment:
Toán tử `&.` có tác dụng gì?

Reply:
Toàn tử `&.` có tác dụng là:
- Kiểm tra xem đối tượng gọi hàm có nil hay ko. Nếu là nil thì nó sẽ return nil. Nếu không nil thì sẽ gửi phương thức đến đối tượng đó.